package.json

```diff
@@ -19,18 +19,18 @@
     "test:ts": "tsc --noEmit -p ./tsconfig.json",
     "test:flow": "flow",
     "test:flow:coverage": "flow-coverage-report",
-    "test:build": "jest --projects other/misc-tests other/react-native",
-    "test:cypress:dev": "npm-run-all --parallel --race storybook cy:open",
-    "pretest:cypress": "npm run storybook:build --silent",
-    "test:cypress": "start-server-and-test storybook:serve http://localhost:6006 cy:run",
+    "test:build": "jest --config other/misc-tests",
+    "// FIXME: test:build": "jest --projects other/misc-tests other/react-native",
```

Comment:
The react native tests are broken. Could a react native maintainer help figure out what's wrong? travis-ci.org/paypal/downshift/builds/437797785

Reply:
Seems like SFCs are treated like Stateful Components because it was looking for `render`...